.eslintrc.yaml

```diff
@@ -135,16 +135,17 @@ rules:
   }]
   no-tabs: error
   no-trailing-spaces: error
+  object-curly-spacing: [error, always]
   one-var-declaration-per-line: error
   operator-linebreak: [error, after]
   quotes: [error, single, avoid-escape]
   semi: error
   semi-spacing: error
   space-before-blocks: [error, always]
   space-before-function-paren: [error, {
-    "anonymous": "never",
-    "named": "never",
-    "asyncArrow": "always"
+    anonymous: never,
```

Comment:
@sebdeckers "bureaucratic" rules have their reasons ;) One of them is that keeping commits logically separated makes backporting a lot easier. Most of the changes from this PR are going to have numerous conflicts with release branches, while this one would have been cherry-picked cleanly.